docs/farming-&-staking/farming/additional-guides/network-info.md

```diff
@@ -0,0 +1,78 @@
+
+## Understanding Port Forwarding
+
+Forwarding ports on your router will open up a specific lane on your router to allow external traffic to flow in or out. This process is usually automatic for common applications, but for specific needs like ours, it must be done manually. For a foundational understanding of port forwarding, you can read this guide: [Port Forwarding Overview](https://www.geeksforgeeks.org/port-forwarding-on-router-and-why-do-we-need-it/).
+
+## Required Ports for Subspace Network
+
+For optimal performance of our blockchain node, specific ports need to be configured to handle incoming connections. These ports facilitate various critical functions within our blockchain network
+
+Below is a table summarizing the necessary port configurations for uninterrupted network performance:
+
+| Port  | Protocol            | Exposure                                                | Short Description           | Used By                                                                                                                                                              | Purpose                                              |
+|-------|---------------------|---------------------------------------------------------|----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
+| 30333 | TCP/UDP (QUIC)      | Always public                                           | Substrate networking P2P   | <ul><li>[Pulsar](../../../category/pulsar/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li><li>[Space Acres](https://github.com/nazar-pc/space-acres)</li></ul> | Connecting to the rest of the network for exchanging blocks, transactions, PoT gossip, etc. |
+| 30433 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Pulsar](../../../category/pulsar/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li><li>[Space Acres](https://github.com/nazar-pc/space-acres)</li></ul> | Connecting to the rest of the network and retrieve archival history            |
+| 30533 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | Connecting to the rest of the network and retrieve archival history, not needed for Pulsar and Space Acres because node and farmer share Subspace networking stack there      |
```

Comment:
```suggestion
| 30533 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Pulsar and Space Acres because the node and the farmer share the Subspace networking stack there. |
```